e2e/test/iothub/device/TwinFaultInjectionTests.cs

```diff
@@ -235,6 +282,7 @@ async Task InitAsync(IotHubDeviceClient deviceClient, TestDevice testDevice)
 
             async Task TestOperationAsync(IotHubDeviceClient deviceClient, TestDevice testDevice)
             {
+                string propName = Guid.NewGuid().ToString();
```

Comment:
https://github.com/Azure/azure-iot-sdk-csharp/blob/69f53aa08d8ee723cc3cef92415c373c01161671/iothub/device/src/Twin/ReportedProperties.cs#L51

Reply:
Property key now needs to be unique